components/events/FollowCalendar.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+  <div class="follow-calendar__section">
+    <h3>Follow our event calendar</h3>
+    <p class="follow-calendar__section__description">
+      <!-- eslint-disable-next-line vue/singleline-html-element-content-newline -->
+      Stay up to date with all of our scheduled events by following our calendar. You can view the calendar by visiting <AppLink v-bind="qiskitCalendarLink">{{ qiskitCalendarLink.label }}</AppLink>, or subscribe to it by adding to the calendar app of your choice.
+    </p>
+    <div class="follow-calendar__tabs">
+      <cv-tabs aria-label="Calendar applications">
+        <cv-tab
+          v-for="{ name, instructions } in calendarsInstructions"
+          :id="`tab-${name}`"
+          :key="name"
+          :label="name"
+          class="follow-calendar__tab"
+        >
+          <p class="follow-calendar__sync">
+            Start by copying the calendar subscription link
+            <cv-code-snippet
+              kind="inline"
+              feedback-aria-label="Copy calendar sync url"
+            >
+              <span>{{ qiskitCalendarSyncLink }}</span>
+            </cv-code-snippet>
+          </p>
+          <component :is="instructions" class="follow-calendar__instructions" />
+        </cv-tab>
+      </cv-tabs>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+import GoogleCalendarInstructions from '~/components/events/calendars/GoogleInstructions.vue'
+import OutlookCalendarInstructions from '~/components/events/calendars/OutlookInstructions.vue'
+import AppleCalendarInstructions from '~/components/events/calendars/AppleInstructions.vue'
+
+import { GeneralLink } from '~/constants/appLinks'
+
+@Component({
+  components: {
+    AppleCalendarInstructions,
+    GoogleCalendarInstructions,
+    OutlookCalendarInstructions
+  }
+})
+
```

Comment:
Delete this empty line. You are using a [class decorator `@Component`](https://class-component.vuejs.org/guide/class-component.html#data) and decorators go always right before the code they are modifying